config/manager/numaflow-controller-definitions-config.yaml

```diff
@@ -50,7 +50,25 @@ data:
                     - name: NUMAFLOW_LEADER_ELECTION_DISABLED
                       valueFrom:
                         configMapKeyRef:
-                          key: controller.disable.leader.election
+                          key: controller.leader.election.disabled
+                          name: numaflow-cmd-params-config
+                          optional: true
+                    - name: NUMAFLOW_LEADER_ELECTION_LEASE_DURATION
+                      valueFrom:
+                        configMapKeyRef:
+                          key: controller.leader.election.lease.duration
+                          name: numaflow-cmd-params-config
+                          optional: true
+                    - name: NUMAFLOW_LEADER_ELECTION_LEASE_RENEW_DEADLINE
+                      valueFrom:
+                        configMapKeyRef:
+                          key: controller.leader.election.lease.renew.deadline
+                          name: numaflow-cmd-params-config
+                          optional: true
+                    - name: NUMAFLOW_LEADER_ELECTION_LEASE_RENEW_PERIOD
+                      valueFrom:
+                        configMapKeyRef:
+                          key: controller.leader.election.lease.renew.period
```

Comment:
any idea why all of the stuff in this file changed?

Reply:
ah because I updated the numaflow controller to match v1.2.1.